content/en/synthetics/explorer/search.md

```diff
@@ -0,0 +1,76 @@
+---
+
+## Overview
+
+After selecting a time frame from the dropdown menu on the top right, you can search for batches of CI jobs in the [CI Results Explorer][1].
+
+## Explore facets
+
+The facets panel on the left lists several facets you can use to search through your batches. To start customizing the search query, click through the list of facets starting with **Batch**.
+
+### Batch attributes
+
+**Batch** facets allow you to filter on your batches' attributes.
+
+| Facet            | Description                                                 |
+|------------------|-------------------------------------------------------------|
+| `Summary Status` | The status of the batch: `Passed`, `Failed`, and `In Progress`. |
+| `Duration`       | The overall duration of the batch.                          |
+| `ID`             | The batch ID.                                               |
+
+### CI attributes
+
+**CI** facets allow you to filter on your batches' CI-related attributes.
+
+| Facet          | Description                                 |
+|----------------|---------------------------------------------|
+| `CI Provider`  | The CI provider associated with the batch.  |
+| `Job Name`     | The job name associated with the batch.     |
+| `Job URL`      | The job URL associated with the batch.      |
+| `Pipeline ID`  | The pipeline ID associated with the batch.  |
+| `Pipeline Name` | The pipeline or repository name associated with the batch. |
+| `Pipeline Number` | The pipeline or build number associated with the batch. |
+| `Pipeline URL` | The pipeline URL associated with the batch. |
+| `Stage Name`   | The stage name associated with the batch.   |
+
+### Test result attributes
+
+**Test result** facets allow you to filter on your executed test results' attributes.
+
+| Facet            | Description                                                                                             |
+|------------------|---------------------------------------------------------------------------------------------------------|
+| `Execution Rule` | The execution rule associated with the test result of the batch: `Blocking`, `Non Blocking`, and `Skipped`. |
+| `Fast Retries`   | The number of fast retries associated with the test result of the batch.                                |
+| `Location`       | The location associated with the test result of the batch.                                              |
+| `Test ID`        | The test ID associated with the test result of the batch.                                               |
+| `Test Name`      | The test name associated with the test result of the batch.                                             |
+
+### Git attributes
+
+**Git** facets allow you to filter on your batches' Git-related attributes.
+
+| Facet       | Description                               |
+|-------------|-------------------------------------------|
+| `Branch`    | The branch associated with the batch.     |
```

Comment:
@alai97 Last minute change, could we also add the `Author Email` facet? We added it quite recently, and it corresponds to the commit author